guide/bitcoin/bitcoin-client.md

```diff
@@ -120,6 +120,12 @@ Expected output:
   > Primary key fingerprint: ...
   ```
 
+* This command must generate a number greater than 0
+
+  ```sh
+  $ gpg --verify SHA256SUMS.asc 2>&1 | grep Good | wc -l
```

Comment:
Why this is useful? I got number 12 nothing more, what does mean this?

Reply:
> Good idea, tACK man page, but what do you mean by "the batch output of gpg verify leads to misunderstandings"?

this is not a problem of this great guide that guide the user through each steps.. very nice.

Unfortunately for newbies the gpg output is a hodgepodge of "WARNING, Good, Sign, emails" confusing words mixed into the same output...